build-logic/uber-plugins/src/main/kotlin/gradlebuild.internal.kotlin-js.gradle.kts

```diff
@@ -50,6 +51,12 @@ rootProject.run {
             lockFileDirectory = layout.buildDirectory.file("kotlin-js-store").get().asFile
         }
     }
+
+    yarnpkgMirrorUrl.orNull?.let { mirrorUrl ->
+        tasks.withType<org.jetbrains.kotlin.gradle.targets.js.npm.tasks.KotlinNpmInstallTask>().configureEach {
+            args += listOf("--registry", mirrorUrl)
```

Comment:
Ah, I thought the Npm install task would pick up some environment variable automatically. Doing it like this is fine as well I suppose.